spec/models/user_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
```

Comment:
@jw9billions please delete this since it is not being used. Devise handles this.